cli/compose/convert/service.go

```diff
@@ -36,9 +37,19 @@ func Services(
 		if err != nil {
 			return nil, errors.Wrapf(err, "service %s", service.Name)
 		}
-		configs, err := convertServiceConfigObjs(client, namespace, service.Configs, config.Configs)
-		if err != nil {
-			return nil, errors.Wrapf(err, "service %s", service.Name)
+		var configs []*swarm.ConfigReference
+		if len(service.Configs) > 0 {
+			version, err := client.ServerVersion(context.Background())
+			if err != nil {
+				return nil, errors.Wrapf(err, "service %s", service.Name)
+			}
+			if versions.LessThan(version.APIVersion, "1.30") {
+				return nil, errors.Errorf("Server API version (%s) doesn't support configs declared in service %s", version.APIVersion, service.Name)
```

Comment:
Can you make this error consistent with the [other errors we have](https://github.com/docker/cli/blob/ed5762cb9f41f373696d7e8fbbfdf4c7af03a621/cmd/docker/docker.go#L259-L269)? 

```
docker service ls requires API version 1.24, but the Docker daemon API version is 1.10
```